kernel-install/50-mkosi.install

```diff
@@ -44,11 +45,14 @@ def mandatory_variable(name: str) -> str:
 
 
 def build_microcode_initrd(output: Path) -> Optional[Path]:
-    amd = Path("/usr/lib/firmware/amd-ucode")
-    intel = Path("/usr/lib/firmware/intel-ucode")
+    vendor, ucode = identify_cpu(Path("/"))
 
-    if not amd.exists() and not intel.exists():
-        logging.debug("/usr/lib/firmware/{amd-ucode,intel-ucode} not found, not adding microcode initrd")
+    if vendor is None:
+        logging.debug("Unable to determine the vendor of your CPU, not adding microcode initrd")
```

Comment:
Warning?

Reply:
Also, "not adding microcode" should be sufficient, no need to say that it's packaged up as an initrd

Reply:
Made it a warning and left out the "initrd" at the end. Just mimicked what was there before. ;)